### ISSUE TITLE
UI bug in search bar on the faker.js website

### ISSUE THREAD

**Issue by Wdyffs** [faker-js/faker#2336]
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

I click in search bar at the top of the page https://fakerjs.dev/api/ 
![2](https://github.com/faker-js/faker/assets/87950230/e9cbcb2f-9b98-4bdd-99b6-507285bcfaf6)

Try to type in search bar, but nothing appears in input
![3](https://github.com/faker-js/faker/assets/87950230/317a14ed-a84f-49ae-926f-500407094716)

I find that searching is working and it is typing on the background in mini-search input
![4](https://github.com/faker-js/faker/assets/87950230/9fd0e6a9-d8e3-45a9-913e-15d5042a139c)

May be that is good working, so sorry then, but I found it confusing. Good Luck guys!!!

### Environment Info

```shell
I dont know where to send this bug, and send to the issue topic
```

### Which module system do you use?

- [ ] CJS
- [ ] ESM

### Used Package Manager

npm

**Comment by matthewmayer** [comment]
What is your browser and operating system?

**Comment by Wdyffs** [comment]
Chrome 115.0.5790.171 (Official Build) (64-bit)
Windows 10

**Comment by ST-DDT** [comment]
I can confirm this bug exists.

**Comment by xDivisionByZerox** [comment]
This might have been introduced in #1774 (?).

**Comment by matthewmayer** [comment]
Yes thats the problem. There are two keydown handlers on the `window`, if you remove the first one (introduced in #1774) the search bar works again

<img width="394" alt="Screenshot 2023-09-25 at 11 45 20" src="https://github.com/faker-js/faker/assets/152770/297689f6-dcd7-48cc-bb1f-bf63e4b9f516">

**Comment by xDivisionByZerox** [comment]
@ST-DDT is this feature required? We could simply revert the change. If you insist on this UX we can probably work around it. Depending on the decision we can add the `good first issue` label.

**Comment by ST-DDT** [comment]
> @ST-DDT is this feature required?

No, it is not.

**Comment by xDivisionByZerox** [comment]
If someone wants to pick this up: This can be solved by reverting #1774.
